### ISSUE TITLE
cannot run on Clojure 1.7.0

### ISSUE THREAD

**Issue by chrisliu529** [hypirion/hello-swearjure#2]
My test env:
$ lein version
Leiningen 2.5.3 on Java 1.7.0_71 Java HotSpot(TM) 64-Bit Server VM

$ lein repl
nREPL server started on port 63926 on host 127.0.0.1 - nrepl://127.0.0.1:63926
REPL-y 0.3.7, nREPL 0.2.10
Clojure 1.7.0
Java HotSpot(TM) 64-Bit Server VM 1.7.0_71-b14

Error messages:
$ lein run
java.lang.Exception: Error loading /Users/chrisliu/dev/hello-swearjure/project.clj
 at leiningen.core.project$read_raw$fn__6478.invoke (project.clj:943)
    leiningen.core.project$read_raw.invoke (project.clj:940)
    leiningen.core.project$read.invoke (project.clj:954)
    leiningen.core.project$read.invoke (project.clj:955)
    leiningen.core.main$_main$fn__6096.invoke (main.clj:386)
    leiningen.core.main$_main.doInvoke (main.clj:383)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.lang.Var.invoke (Var.java:379)
    clojure.lang.AFn.applyToHelper (AFn.java:154)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:630)
    clojure.main$main_opt.invoke (main.clj:316)
    clojure.main$main.doInvoke (main.clj:421)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
Caused by: clojure.lang.Compiler$CompilerException: clojure.lang.ArityException: Wrong number of args (1) passed to: core/bit-clear, compiling:(/Users/chrisliu/dev/hello-swearjure/project.clj:1:1)
 at clojure.lang.Compiler.load (Compiler.java:7239)
    clojure.lang.Compiler.loadFile (Compiler.java:7165)
    clojure.lang.RT$3.invoke (RT.java:319)
    leiningen.core.project$read_raw$fn__6478.invoke (project.clj:941)
    leiningen.core.project$read_raw.invoke (project.clj:940)
    leiningen.core.project$read.invoke (project.clj:954)
    leiningen.core.project$read.invoke (project.clj:955)
    leiningen.core.main$_main$fn__6096.invoke (main.clj:386)
    leiningen.core.main$_main.doInvoke (main.clj:383)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.lang.Var.invoke (Var.java:379)
    clojure.lang.AFn.applyToHelper (AFn.java:154)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:630)
    clojure.main$main_opt.invoke (main.clj:316)
    clojure.main$main.doInvoke (main.clj:421)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
Caused by: clojure.lang.ArityException: Wrong number of args (1) passed to: core/bit-clear
 at clojure.lang.AFn.throwArity (AFn.java:429)
    clojure.lang.AFn.invoke (AFn.java:32)
    clojure.lang.Var.invoke (Var.java:379)
    leiningen.core.project$eval603$fn__604.doInvoke (project.clj:2)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    leiningen.core.project$eval603.invoke (project.clj:2)
    clojure.lang.Compiler.eval (Compiler.java:6782)
    clojure.lang.Compiler.load (Compiler.java:7227)
    clojure.lang.Compiler.loadFile (Compiler.java:7165)
    clojure.lang.RT$3.invoke (RT.java:319)
    leiningen.core.project$read_raw$fn__6478.invoke (project.clj:941)
    leiningen.core.project$read_raw.invoke (project.clj:940)
    leiningen.core.project$read.invoke (project.clj:954)
    leiningen.core.project$read.invoke (project.clj:955)
    leiningen.core.main$_main$fn__6096.invoke (main.clj:386)
    leiningen.core.main$_main.doInvoke (main.clj:383)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.lang.Var.invoke (Var.java:379)
    clojure.lang.AFn.applyToHelper (AFn.java:154)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:630)
    clojure.main$main_opt.invoke (main.clj:316)
    clojure.main$main.doInvoke (main.clj:421)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)

**Comment by hypirion** [comment]
Yeah, I think you need exactly a lein version that uses the clojure version version 2.1.0 uses. I'll see if I can fix this, should just be some integer replacements.
